docs/BreakingChanges/102 ListBoxItem IsSelected Binding Issue with Observab.md

```diff
@@ -0,0 +1,35 @@
+## 102: ListBoxItem IsSelected Binding Issue with ObservableCollection<T>.Move
```

Comment:
{
  "Id": "102",
  "Title": "ListBoxItem IsSelected Binding Issue with ObservableCollection<T>.Move",
  "Details": "Calling `ObservableCollection<T>.Move` on a collection bound to a ListBox with items selected can lead to erratic behavior with future selection or unselection of ListBox items",
  "Suggestion": "Calling `ObservableCollection<T>.Remove` and `ObservableCollection<T>.Insert` instead of `ObservableCollection<T>.Move` will work around this issue. Alternatively, this issue has been fixed in the .NET Framework 4.6 and may be addressed by upgrading to that version of the .NET Framework.",
  "Link": "http://social.msdn.microsoft.com/Forums/en-US/afcbc8b3-a2f2-41e4-b402-2efc9eab1ffe/listboxitem-isselected-binding-issue-with-observablecollectiontmove?forum=wpf",
  "Markdown": null,
  "ApplicableApis": [
    "M:System.Collections.ObjectModel.ObservableCollection`1.Move(System.Int32,System.Int32)",
    "M:System.Collections.ObjectModel.ObservableCollection`1.MoveItem(System.Int32,System.Int32)"
  ],
  "Related": null,
  "VersionBroken": "4.5",
  "VersionFixed": "4.6",
  "IsBuildTime": false,
  "IsQuirked": false,
  "IsSourceAnalyzerAvailable": false,
  "BugLink": "http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/999613",
  "Notes": "Source analyzer status: Pri 2, Planned - No tracking work item yet",
  "IsRetargeting": false,
  "ImpactScope": 2
}

**Parsed APIs:**
- [x] M:System.Collections.ObjectModel.ObservableCollection`1.Move(System.Int32,System.Int32)
- [x] M:System.Collections.ObjectModel.ObservableCollection`1.MoveItem(System.Int32,System.Int32)